modal/cli/volume.py

```diff
@@ -244,7 +244,7 @@ def _destination_stream():
         else:
             with NamedTemporaryFile(delete=False) as fp:
                 yield fp
-            Path(fp.name).rename(destination)
+            shutil.move(fp.name, destination)
```

Comment:
Alas there are no existing tests, and not obvious how to easily test :see_no_evil: